### PR TITLE
Update pricing for Infrastructure Monitoring

### PR DESCRIPTION
**Pull request by cpipilas**
SSIA

### CODE REVIEW

File: docs/monitoring/spm-faq.md [hunk 1]
```diff
@@ -1063,16 +1063,16 @@ Note:
...
 If you send metrics from a server A to Monitoring App Foo between 01:00 and
-02:00 that's $0.035 for the Standard plan.
+02:00 that's $0.005 for the Standard plan.
 
 If another agent is monitoring something else, even if that is running on the same
-server A, and sending metrics to a different Monitoring App Bar, that's another $0.035.
+server A, and sending metrics to a different Monitoring App Bar, that's another $0.005.
 
 If you are not sending metrics from a server A for a Monitoring App Foo between
 02:00 and 03:00 then you pay $0 for that hour.
```
Comment (otisg) [inline docs/monitoring/spm-faq.md:1073]: @cpipilas Below 2 lines are correct for *Infra* (but not Integrations) monitoring.

File: docs/monitoring/spm-faq.md [hunk 2]
```diff
@@ -1063,16 +1063,16 @@ Note:
 Usage is metered hourly on a per-agent basis. For example:
 
 If you send metrics from a server A to Monitoring App Foo between 01:00 and
-02:00 that's $0.035 for the Standard plan.
+02:00 that's $0.005 for the Standard plan.
```
Comment (otisg) [inline docs/monitoring/spm-faq.md:1066]: Retention affects the cost, too, not just the plan.  So Standard plan ==> Standard plan & 7 days retention.

File: docs/monitoring/spm-faq.md (hunk 1 above)
Comment (cpipilas) [reply docs/monitoring/spm-faq.md:1078]: Updated to include both Infra & Integrations.

File: docs/monitoring/spm-faq.md (hunk 2 above)
Comment (cpipilas) [reply docs/monitoring/spm-faq.md]: Added more details about retention.

**Review by otisg — APPROVED** [review]

**Pull request merged**